### ISSUE TITLE
Invalid redirection on report error

### ISSUE THREAD

**Issue by donarus** [openanalytics/shinyproxy#149]
We have the following settings in the config YAML file:
```
server:
  contextPath: /research-apps
  servlet:
    context-path: /research-apps
```

Then, when a user reports an issue, the back redirection is invalid. This issue occurs only when running on different context path than **/**

![image](https://user-images.githubusercontent.com/1846145/58005967-2c40f780-7ae7-11e9-9067-d33ed2117a72.png)
------
![image](https://user-images.githubusercontent.com/1846145/58005737-9c9b4900-7ae6-11e9-9b5e-c2ec889d6265.png)
------
![image](https://user-images.githubusercontent.com/1846145/58005691-7fff1100-7ae6-11e9-959a-43d52bd740ec.png)

It seems that the root cause is here: https://github.com/openanalytics/shinyproxy/blob/master/src/main/java/eu/openanalytics/shinyproxy/controllers/IssueController.java#L68

**Comment by donarus** [comment]
Also, I'm not a big fan of redirecting in case an error occurs (see #63). What I'm thinking about is to add the option 
"redirectOnErrorReport" (by default TRUE), but when set FALSE, no redirect could occur.

something like:

```
proxy:
  support:
    mail-to-address: tpalusga@xxxxx.com
    mail-from-address: issue@xxxxx.com
    redirectOnErrorReport: <<TRUE/FALSE>> 
```   

What do you think about it? It would be easy to implement and I could prepare the PR if we decide it is worth implementing.

**Comment by jat255** [comment]
I just noticed we're experiencing this same issue. @donarus did you ever implement this change for your own use?

**Comment by LEDfan** [comment]
Hi, this will be fixed in the next release.
The form is now a client-side only form. So after submitting the form, the modal is closed and the user stays on the same page (without redirect or reload).

**Comment by LEDfan** [comment]
This is now fixed in [ShinyProxy 2.6.0](https://shinyproxy.io/downloads/#260) (ContainerProxy 0.8.10).

Thank you for reporting this issue.
